### PR TITLE
Extend Change Date to June 15th

### PR DESCRIPTION
**Pull request by SteveLasker**
Signed-off-by: Steve Lasker <stevelasker@hotmail.com>

### CODE REVIEW

**Comment by SteveLasker** [comment]
@bariscaglar, @toddysm

File: client-firewall-rules.md [hunk 1]
```diff
@@ -3,38 +3,110 @@
...
 | Purpose | Protocol | Target FQDN | Available |
 | - | - | - | - |
 | Registry Endpoint | https | `mcr.microsoft.com` | Now |
-| Data Endpoint | https | `*.cdn.mscr.io` | Now - April 20, 2020 |
-| Data Endpoint | https | `*.data.mcr.microsoft.com` | Starting April  13, 2020 |
+| Data Endpoint | https | `*.cdn.mscr.io` | Now |
+| Data Endpoint | https | `*.data.mcr.microsoft.com` | Starting June  15, 2020 |
 
-If you're updating this prior to April 13, adding both date endpoints will assure a smooth transition. Once the consolidated data endpoint deployment is complete, we'll update this page.
+If you're updating this prior to June 15th, adding both date endpoints will assure a smooth transition. Once the consolidated data endpoint deployment is complete, we'll update this page enabling the removal of `*.cdn.mscr.io`.
```
Comment (toddysm) [inline client-firewall-rules.md]: You have a spelling error: "adding both date endpoints" should read "adding both data endpoints"

File: client-firewall-rules.md [hunk 2]
```diff
@@ -3,38 +3,110 @@
...
+| Data Endpoint | https | `*.data.mcr.microsoft.com` | Starting June  15, 2020 |
 
-If you're updating this prior to April 13, adding both date endpoints will assure a smooth transition. Once the consolidated data endpoint deployment is complete, we'll update this page.
+If you're updating this prior to June 15th, adding both data endpoints will assure a smooth transition. Once the consolidated data endpoint deployment is complete, we'll update this page enabling the removal of `*.cdn.mscr.io`.
 
 ![Azure Application Rule](./media/mcr-client-firewall-rules.png)
 
+## Testing the `*.cdn.mscr.io` Data Endpoint
+
+To validate configurations pre March 3rd, 2020, the following multi-arch images are available for testing.
```
Comment (bariscaglar) [inline client-firewall-rules.md]: March 3rd date is no longer correct. The test images is already out to be used.

File: client-firewall-rules.md [hunk 3]
```diff
@@ -3,38 +3,110 @@
...
+`curl https://mcr.microsoft.com/v2/hello-world/blobs/sha256:fce289e99eb9bca977dae136fbe2a82b6b7d4c372474c9235adc1741675f587e`
+
+produces a url with `[oldRegionReference].cdn.mscr.io` :
+```
+<a href="https://mcrwcus0.cdn.mscr.io/f439446e126348d29d8a8c39253da1ca-gqcvpppbjg//docker/registry/v2/blobs/...
+```
+
+## Testing the `*.data.mcr.microsoft.com` Data Endpoint
+
+To validate changes will work post March 3rd, 2020, the following images are available for testing today.
```
Comment (bariscaglar) [inline client-firewall-rules.md]: Same date here

File: client-firewall-rules.md [hunk 4]
```diff
@@ -3,38 +3,110 @@
...
+`curl  https://mcr.microsoft.com/v2/mcrlabscdnone/hello-world-linux/blobs/sha256:fce289e99eb9bca977dae136fbe2a82b6b7d4c372474c9235adc1741675f587e`
+
+produces a url with `[region].data.mcr.microsoft.com` :
+```
+<a href="https://westcentralus.data.mcr.microsoft.com/f439446e126348d29d8a8c39253da1ca-gqcvpppbjg//docker/registry/v2/blobs/...
+```
+
+### Q&A
+- **Q: What do I do now, to prepare for this change?**  
+  **A:** Follow the above guidance, adding both data endpoints.
```
Comment (bariscaglar) [inline client-firewall-rules.md]: to your outbound firewall rules. If you do not use outbound firewall rules, no change is necessary.

File: client-firewall-rules.md [hunk 5]
```diff
@@ -3,38 +3,110 @@
...
+<a href="https://westcentralus.data.mcr.microsoft.com/f439446e126348d29d8a8c39253da1ca-gqcvpppbjg//docker/registry/v2/blobs/...
+```
+
+### Q&A
+- **Q: What do I do now, to prepare for this change?**  
+  **A:** Follow the above guidance, adding both data endpoints. 
+- **Q: Are there any risks to adding both endpoints?**  
+  **A:** Both data endpoints are owned and maintained by Microsoft. Both are secure. Adding both domains eases the transition.
+- **Q: If both domains are secure, why are you making this change?**  
+  **A:** We've received feedback that having multiple root domains is confusing as it's not obvious that registries require multiple endpoints. Putting both endpoints under `microsoft.com` provides more trust and confidence these are operated by Microsoft.
```
Comment (bariscaglar) [inline client-firewall-rules.md]: confidence that

**Pull request merged**